<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChapterCitationWindow</class>
 <widget class="QMainWindow" name="ChapterCitationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input data for citation</string>
    </property>
    <widget class="QLabel" name="label_author">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_year">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_author">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_year">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_title">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>278</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="radio_layout">
     <item>
      <widget class="QRadioButton" name="radio_1st">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>One Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_2nd">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2nd Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_3rd">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3rd Author</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_in">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input data for book where chapter is included!</string>
    </property>
    <widget class="QLabel" name="label_title_in">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>20</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_year_in">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_author_in">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>147</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_place_in">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Place</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_place_in">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_year_in">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>136</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_publisher_in">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>146</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_publisher_in">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Publisher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineedit_title_in">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>134</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_author_in">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>33</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>278</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="radio_layout_2">
     <item>
      <widget class="QRadioButton" name="radio_1st_in">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>One Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_2nd_in">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2nd Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_3rd_in">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3rd Author</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#000000;
}

QPushButton:Pressed{
background-color:#24c8c8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>168</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="button_copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_go">
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_copy">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>R
e
s
e
t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset_in">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>R
e
s
e
t</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineedit_author</tabstop>
  <tabstop>lineedit_year</tabstop>
  <tabstop>lineedit_title</tabstop>
  <tabstop>lineedit_author_in</tabstop>
  <tabstop>lineedit_year_in</tabstop>
  <tabstop>lineedit_title_in</tabstop>
  <tabstop>lineedit_place_in</tabstop>
  <tabstop>lineedit_publisher_in</tabstop>
  <tabstop>button_go</tabstop>
  <tabstop>radio_1st</tabstop>
  <tabstop>radio_2nd</tabstop>
  <tabstop>radio_3rd</tabstop>
  <tabstop>radio_1st_in</tabstop>
  <tabstop>radio_2nd_in</tabstop>
  <tabstop>radio_3rd_in</tabstop>
  <tabstop>button_copy</tabstop>
  <tabstop>button_back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
